6.X/6.4.1/01-command_line_arguments.md

```diff
@@ -0,0 +1,192 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Command Line Arguments -->
+<!-- Layout: plain -->
+
+# Command line arguments explained
+
+<br />
+
+## Command: `deploy`
+
+`--skip-namespace`
+
+- installer does not deploy the `namespace.yaml` manifest.
+  It expects the Namespace to exist and to match the value in `values.yaml`
+  There is no validation, in case of mismatch the installer will fail
```

Comment:
remove

Reply:
What exactly should I remove?

Reply:
That line is added to the previous statement, isn't it?

Reply:
Done